src/modules/silentpayments/Makefile.am.include

```diff
@@ -0,0 +1,2 @@
+include_HEADERS += include/secp256k1_silentpayments.h
+noinst_HEADERS += src/modules/silentpayments/main_impl.h
```

Comment:
in "build: add skeleton for new silentpayments (BIP352) module" (https://github.com/bitcoin-core/secp256k1/pull/1471/commits/a9a5fe8e28f2f4dd53fdfb4ce797b23b4452f0a6):

Needed for Bitcoin Core's CI:
```suggestion
noinst_HEADERS += src/modules/silentpayments/tests_impl.h
noinst_HEADERS += src/modules/silentpayments/main_impl.h
```